agent/app/agent_capability.go

```diff
@@ -498,6 +503,19 @@ func (agent *ecsAgent) appendServiceConnectCapabilities(capabilities []*ecs.Attr
 	return capabilities
 }
 
+func (agent *ecsAgent) appendGuardDutyCapabilities(capabilities []*ecs.Attribute) []*ecs.Attribute {
+	supported, _ := isGuardDutySupported()
+	if !supported {
+		seelog.Debugf("GuardDuty is not supported. Skipping appendGuardDutyCapabilities.")
+		return capabilities
+	}
```

Comment:
this check is redundant since we're appending the attribute only if supported

Reply:
removed this